test/e2e/docker_test.go

```diff
@@ -851,10 +851,9 @@ func TestDockerKubernetes126to127GithubFluxEnabledUpgradeFromLatestMinorRelease(
 
 func TestDockerKubernetes126WithOIDCManagementClusterUpgradeFromLatestSideEffects(t *testing.T) {
 	provider := framework.NewDocker(t)
-	runTestManagementClusterUpgradeSideEffects(t, provider, "", v1alpha1.Kube126, api.ClusterToConfigFiller(
+	runTestManagementClusterUpgradeSideEffects(t, provider, framework.OS(""), v1alpha1.Kube126, api.ClusterToConfigFiller(
```

Comment:
nit: we could have a `Docker` OS
Which would be ignored when creating the config fillers, same way as it is now, but at least it represents the OS better than just mpety string